src/smpp_gateway/outgoing.py

```diff
@@ -48,5 +48,6 @@ def send(self, id_, text, identities, context=None):
             MTMessage.objects.bulk_create(
                 [MTMessage(**kwargs) for kwargs in kwargs_group]
             )
-            with connection.cursor() as curs:
-                curs.execute(f"NOTIFY {self.model.name}")
+            # FIXME: Re-enable when we have a way to avoid notifies on bulk SMS
```

Comment:
@simonkagwi In addition to adding `--event-loop-timeout` (default 5 seconds), want to try addressing this comment in this branch?

Reply:
@tobiasmcnulty  Yeah, sure. I'll do that